worker/src/handlers/aggregationProcessor/aggregationProcessor.js

```diff
@@ -366,18 +367,21 @@ const aggregationProcessor = async (
   const addPipelines = getAddPipelines({ model, now, actualNow });
   const subtractPipeline = getSubtractPipeline({ model, now });
 
-  const addResultsPromise = addPipelines.map((addPipeline) => {
+  const addResultsPromise = addPipelines.map((addPipeline, key) => {
     const out = Statement.aggregate(addPipeline).option({
       maxTimeMS: AGGREGATION_TIMEOUT_MS,
       readPreference
-    });
+    }).hint(
+      (key === 0 ?
+        { organisation: 1, timestamp: -1, _id: 1 } : { organisation: 1, stored: -1, _id: 1 })
```

Comment:
Mind making these two index shapes constants please? Then we can use it below
```
INDEX_ORG_1_TIMETSTAMP_-1__ID_1 = { organisation: 1, timestamp: -1, _id: 1 }
INDEX_ORG_1_STORED_-1__ID_1 = { organisation: 1, stored: -1, _id: 1 }
```

We might then look to use these constants to validate the indexes exists in a migration down the line. The queries will literally fail if they aren't there